### ISSUE TITLE
PPA Management and Testing

### ISSUE THREAD

**Issue by doctormo** [DIGImend/digimend-kernel-drivers#100]
Here is the new PPA for older ubuntu releases:

https://launchpad.net/~digimend/+archive/ubuntu/releases

So far I've only pushed 16.04 (LTS) but will do more as testers become available.

**Comment by spbnick** [comment]
Awesome, thank you! Is it the latest master, though, or the actual v6 release?

**Comment by doctormo** [comment]
Ah yes, you're right, it's dev. You've got your debian files committed to your main repository which makes this a bit harder to do for the tagged release, but I'll sort it out going forwards.

**Comment by spbnick** [comment]
Yeah, I know, having debian files in the repo makes it more difficult for packagers. However as far as I do my own packaging I'll need to keep them somewhere too. We can work on that, though.

One proper fix for this would be updating the debian files, and all other files mentioning the version number to v7. This should've been done immediately after v6 release, but I probably forgot. Can you do that? Then you'll have your PPA version correct, although you'll need to mark it as pre-release somehow (with something like git hash in the version?).

**Comment by doctormo** [comment]
Yes I can do that.

The recommended build process is `git pull; make release; run fancy script to take tarball and put existing debian file in a fresh directory; debuild` but most upstreams find that tedious compared to debuild from git directory. I notice there's no make release for digimend anyway, so moot point.

**Comment by spbnick** [comment]
We can add "make release" if that would help.

**Comment by doctormo** [comment]
It would be something debian people would want to see I think. But I can work around the unclean tarball problem (if you do add it, make sure not to include the debian directory)

**Comment by spbnick** [comment]
Yes, of course. Filed #101.

**Comment by doctormo** [comment]
I've reset the v6 package to the v6 code (as tagged) and created a digimend-kernel-git package which has the latest version. We can't use the v7 version directly as launchpad will check the contents of the tarball for the same version and reject tarballs that change but claim to be the same version. So I'm using the date as the version.

**Comment by spbnick** [comment]
OK, great, thanks :)

**Comment by thwaller** [comment]
Will this PPA be updated for use with 18.04, the latest LTS Ubuntu? For such users, do you recommend using the PPA, or using a local install of the deb?

**Comment by spbnick** [comment]
@doctormo, I guess @thwaller's question is for you to answer :)
